tests/correctness/exp/any-base-2.expect

```diff
@@ -1,5 +1,5 @@
 {
-  "cycles": 257,
+  "cycles": 259,
```

Comment:
Why did the cycle counts of things go up?

Reply:
I think the main reason is that the python frontend/Dahlia frontend don't use `static<>` annotations yet